std/concurrency.d

```diff
@@ -1716,15 +1716,15 @@ void yield(T)(T value)
 }
 
 
-version (Win64)
+/*version (Win64)
 {
     // fibers are broken on Win64
 }
 else version (Win32)
 {
     // fibers are broken in Win32 under server 2012: bug 13821
 }
-else unittest
+else */unittest
```

Comment:
Hopefully we can remove this after the test completes on the Windows autotester.